src/contract.ts

```diff
@@ -103,36 +103,39 @@ export class Contract {
                 })
             });
         });
-        changeMethods.forEach((methodName) => {
-            Object.defineProperty(this, methodName, {
-                writable: false,
-                enumerable: true,
-                value: nameFunction(methodName, async (...args: any[]) => {
-                    if (args.length && (args.length > 3 || !(isObject(args[0]) || isUint8Array(args[0])))) {
-                        throw new PositionalArgsError();
-                    }
+        changeMethods.forEach((baseMethodName) => {
+            ['', 'Raw'].forEach((resultType) => {
+                const methodName = `${baseMethodName}${resultType}`;
+                Object.defineProperty(this, methodName, {
+                    writable: false,
+                    enumerable: true,
+                    value: nameFunction(methodName, async (...args: any[]) => {
+                        if (args.length && (args.length > 3 || !(isObject(args[0]) || isUint8Array(args[0])))) {
+                            throw new PositionalArgsError();
+                        }
 
-                    if(args.length > 1 || !(args[0] && args[0].args)) {
-                        const deprecate = depd('contract.methodName(args, gas, amount)');
-                        deprecate('use `contract.methodName({ args, gas?, amount?, callbackUrl?, meta? })` instead');
-                        return this._changeMethod({
-                            methodName,
-                            args: args[0],
-                            gas: args[1],
-                            amount: args[2]
-                        });
-                    }
+                        if(args.length > 1 || !(args[0] && args[0].args)) {
+                            const deprecate = depd('contract.methodName(args, gas, amount)');
+                            deprecate('use `contract.methodName({ args, gas?, amount?, callbackUrl?, meta? })` instead');
+                            return this[resultType === '' ? '_changeMethod' : '_changeMethodRaw']({
```

Comment:
Small nit that this line is repeated below and could be refactored above to select the property.